packages/storage/src/indexed-db/index.ts

```diff
@@ -138,6 +138,14 @@ export class IndexedDb implements IndexedDbInterface {
     return this.db.get('TRANSACTIONS', source.inner);
   }
 
+  clear() {
```

Comment:
This is a good candidate for a test. Just initialize, assert the tables exist, then call clear and assert they are all gone.

Also, because indexeddb requires a browser and does not exist in node, you'll have to use `pnpm run run-browser-tests` in package.json to run it.

Reply:
@grod220 
```

 it('object store should be empty after clear', async () => {
      const db = await IndexedDb.initialize(testInitialProps);
      await db.saveSpendableNote(newNote);
      expect((await db.getAllNotes()).length).toBe(1);

      await db.saveAssetsMetadata(denomMetadataA);
      expect((await db.getAllAssetsMetadata()).length).toBe(1);

      const tx = {
        blockHeight: 10000n,
        id: new NoteSource(),
        tx: new Transaction(),
        perspective: new TransactionPerspective(),
        view: new TransactionView(),
      };

      await db.saveTransactionInfo(tx);
      expect((await db.getAllTransactions()).length).toBe(1);

      await db.clear();
      expect((await db.getAllNotes()).length).toBe(0);
      expect((await db.getAllAssetsMetadata()).length).toBe(0);
      expect((await db.getAllTransactions()).length).toBe(0);
    });
```

Will it not be enough?

Reply:
we will add other objects as they are realized!

Reply:
Yes, that's fine

Reply:
but I can't check if there are values for some store, (TREE_LAST_POSITION, TREE_LAST_FORGOTTEN ....) for them there is no get implemented... function (or I should check getStateCommitmentTree for this data)

Reply:
No, what you have there is enough I'd say